src/window/building/common.c

```diff
@@ -56,11 +56,26 @@ int window_building_get_vertical_offset(building_info_context *c, int new_window
 void window_building_draw_employment(building_info_context *c, int y_offset)
 {
     building *b = building_get(c->building_id);
+    int text_id = draw_employment_info(c, b, y_offset, 1);
+    draw_employment_details(c, y_offset, text_id, b);
+}
+
+void window_building_draw_employment_without_house_cover(building_info_context *c, int y_offset)
+{
+    building *b = building_get(c->building_id);
+    int text_id = draw_employment_info(c, b, y_offset, 0);
+    draw_employment_details(c, y_offset, text_id, b);
+}
+
+int draw_employment_info(building_info_context *c, building *b, int y_offset, int consider_house_covering)
```

Comment:
Make this function `static` and move it above the other two so you don't have to add a function prototype.

Reply:
done